recipes/kakscalculator2/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin
+make CC=${CXX}
```

Comment:
```suggestion
make CC=${CXX} CFLAGS="${CXXFLAGS}"
```
To get the standard set of compiler flags we use.